packages/esm-bed-management-app/translations/es.json

```diff
@@ -60,7 +60,8 @@
   "required": "Requerido",
   "save": "Guardar",
   "selectBedLocation": "Seleccione una ubicación para la cama",
+  "summary": "Summary",
   "viewBeds": "Ver camas",
-  "wardAllocation": "Asignación de sala",
+  "wardAllocation": "Ward allocation",
```

Comment:
@ibacher  Is it normal that this string got updated because the default translation string got updated?

Reply:
Yes, this is why we need to be _very_ careful when updating the default translation string (Transifex is ultimately dumb, it can't tell if "Asignación de sala" is a good translation for both "Ward Allocation" and "Ward allocation", so it assumes that if the "baseline" translation is changing, the translation string needs to be updated). This needs to be fixed in Transifex before this is mergable.

Reply:
Fixed upstream. Let's see what Transifex sends back.

Reply:
FWIW, it's possible to manually trigger the job that pulls in changes.